### PR TITLE
Add update to dockerless CLI doc and meeting minutes

### PR DESCRIPTION
**Pull request by sigalsax**
This PR updates the Conjur Dockerless CLI with meeting minute, feedback, and result from ruby-packer mini-spike

### CODE REVIEW

**Review by Tovli — DISMISSED** [review]
LGTM

**Review by eladkug — DISMISSED** [review]
LGTM

File: design/proposals/spike-cli-dockerless.md [hunk 1]
```diff
@@ -42,7 +42,21 @@ The CLI design should answer the following requirements:
...
+
+​	Get/Set secret values (known as `variable values <var_id>` functionalities. 
+
+Open for discussion: 
+
+​	List all resources that I have "read" permissions on (known as `list`)
+
+​	Show resource metadata/annotations (known as `show`)
+
+​	Rotate user and host API key (known as `rotate_api_key`)
```
Comment (izgeri) [inline design/proposals/spike-cli-dockerless.md:59]: Should "Check permissions on a resource for a given user/host" also be here? I use this in troubleshooting integrations a lot.

Comment (izgeri) [reply design/proposals/spike-cli-dockerless.md:59]: known as `resource permitted_roles`

Comment (sigalsax) [reply design/proposals/spike-cli-dockerless.md:59]: which one is this in reference to? Rn it seems like you are directly referencing rotate_api_key?

**Review by orenbm — APPROVED** [review]

File: design/proposals/spike-cli-dockerless.md (hunk 1 above)
Comment (izgeri) [reply design/proposals/spike-cli-dockerless.md:59]: I am suggesting a new addition to this list, I just added the comment on the last item

**Pull request merged**